mne/viz/topomap.py

```diff
@@ -171,24 +178,38 @@ def _prepare_topomap_plot(inst, ch_type, sphere=None):
             # change names so that vectorview combined grads appear as MEG014x
             # instead of MEG0142 or MEG0143 which are the 2 planar grads.
             ch_names = [ch_names[k][:-1] + "x" for k in range(0, len(ch_names), 2)]
-        else:
-            assert ch_type in _fnirs_types
-            # Modify the nirs channel names to indicate they are to be merged
+        elif modality == "fnirs":
+            # Modify the channel names to indicate they are to be merged
             # New names will have the form  S1_D1xS2_D2
             # More than two channels can overlap and be merged
             for set_ in overlapping_channels:
                 idx = ch_names.index(set_[0][:-4])
                 new_name = "x".join(s[:-4] for s in set_)
                 ch_names[idx] = new_name
+        elif modality == "opm":
+            # Modify the channel names to indicate they are to be merged
+            # New names will have the form  S1xS2
```

Comment:
Looks like it would be `S1.S2`, no? And maybe the overlapping ones should just end with `OPM-MERGE-REMOVE` or something, and the ones that we keep should just have the original channel names. Then to figure out which ones to remove you do a `.endswith("OPM-MERGE-REMOVE")` (which seems way safer than checking for a ` x ` or `.` in the name, which could happen in real data) and keep the original names.